main/solution/post-deployment/config/settings/.defaults.yml

```diff
@@ -21,7 +21,7 @@ websiteUrl: ${cf:${self:custom.settings.infrastructureStackName}.WebsiteUrl}
 # If a Cognito user pool is setup for the solution, this setting indicates whether native
 # Cognito users should be used or if the Cognito user pool will only be used to federate to other identity providers
 # IMPORTANT: We recommend keeping this flag value set to true when installing SWB for the very first time.
-enableNativeUserPoolUsers: true
+enableNativeUserPoolUsers: false
```

Comment:
In your staging file, you can add `enableNativeUserPoolUsers: false`. You don't need to make changes to the default yml